### PR TITLE
Changes required for dotnet 6 upgrade 

### PR DESCRIPTION
**Pull request by hsubramanianaks**
1. This PR is required for dotnet 6.0 upgrade for vs code to work properly. 
2. This will be tested through and merged. 
3. This can go after CLI PR https://github.com/Azure/Bridge-To-Kubernetes/pull/48 merged and released to production.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -247,6 +247,6 @@
         "url": "https://github.com/Azure/vscode-bridge-to-kubernetes"
     },
     "extensionMetadata": {
-        "expectedCLIVersion": "1.0.20220816.2"
+        "expectedCLIVersion": "1.0.20221103.4"
```
Comment (hsubramanianaks) [inline package.json:250]: this is temporary only for development purposes to test dotnet 6 upgrade changes.

**Comment by elenavillamil** [comment]
Can we update the change log file as well? We should add note there that going back to previous version will need manual action from user.

File: package.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
     "name": "mindaro",
     "displayName": "Bridge to Kubernetes",
     "description": "Rapid Kubernetes development for teams",
-    "version": "1.0.120221031",
+    "version": "1.0.20221219",
```
[the remark this answers is not in the document]
Comment (hsubramanianaks) [reply package.json:5]: Thanks, good catch, it is missing a digit 1.

[the remark this answers is not in the document]
Comment (elenavillamil) [reply package.json]: since we are calling this a major release, should we do 2.0.20221210?

**Review by elenavillamil — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,11 @@
 # Change Log
 
+## [1.0.120230112]
```
Comment (elenavillamil) [inline CHANGELOG.md:3]: Can we update this version to match package.json?

Comment (hsubramanianaks) [reply CHANGELOG.md:3]: Actually package json has to be updated if it is not same version as change log. It's the date we release Vs code.

Comment (hsubramanianaks) [reply CHANGELOG.md:3]: Thanks for catching this and if you don't mind can you update them as well ?

**Pull request merged**